src/tokens-studio/spectrum2-non-colors/spectrum2/layout.component/desktop.json

```diff
@@ -819,6 +829,226 @@
       }
     }
   },
+  "avatar-size-800": {
+    "value": "52px",
+    "type": "sizing",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "avatar-size-800"
+      }
+    }
+  },
+  "avatar-size-900": {
+    "value": "56px",
+    "type": "sizing",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "avatar-size-900"
+      }
+    }
+  },
+  "avatar-size-1000": {
+    "value": "64px",
+    "type": "sizing",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "avatar-size-1000"
+      }
+    }
+  },
+  "avatar-size-1100": {
+    "value": "72px",
+    "type": "sizing",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "avatar-size-1100"
+      }
+    }
+  },
+  "avatar-size-1200": {
+    "value": "80px",
+    "type": "sizing",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "avatar-size-1200"
+      }
+    }
+  },
+  "avatar-size-1300": {
+    "value": "88px",
+    "type": "sizing",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "avatar-size-1300"
+      }
+    }
+  },
+  "avatar-size-1400": {
+    "value": "96px",
+    "type": "sizing",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "avatar-size-1400"
+      }
+    }
+  },
+  "avatar-size-1500": {
+    "value": "104px",
+    "type": "sizing",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "avatar-size-1500"
+      }
+    }
+  },
+  "avatar-group-size-50": {
+    "value": "{avatar-size-50}",
```

Comment:
@GarthDB I feel these should point to the `avatar-size...` token instead of a pixel value, since these match the avatar sizes. Are you OK with this change? I can ping Kami (who worked on these) to confirm as well and update accordingly in the token spec file.

Reply:
I also think the avatar group tokens should point to the `avatar-size...` tokens.

Reply:
I think this is good.